librz/core/cbin.c

```diff
@@ -673,54 +669,89 @@ static bool io_create_mem_map(RzIO *io, RzBinSection *sec, ut64 at) {
 		}
 		return false;
 	}
-	// let the section refere to the map as a memory-map
-	free(iomap->name);
-	iomap->name = sec->map_name && !sec->size ? strdup(sec->map_name) : rz_str_newf("mmap.%s", sec->name);
+	// update the io map's name to refer to the bin map
+	if (map->name) {
```

Comment:
Does it mean that maps can be without a name? And, is it ok to have `iomap` without a name?

Reply:
Yes, I think we should allow names of both kinds of maps to be optional. I've added `RZ_NULLABLE`s now.